app/components/Notifier/notifier.css

```diff
@@ -8,3 +8,13 @@
 
     font-size: 1.4rem;
 }
+
+.hide
```

Comment:
If we're going to have a general class for hiding content, I think `hidden` is a better name.

But if we're doing _that _ it's not so normal for it to be shown (yet still have this class... [via some css trickery]).

So I think we should rename these classes. We can be more specific (even if it needs to be in app.global for now).

maybe: `more_details_button` and `more_details_details`? But that could be too specific... `info_box_details` and `info_box_expander` might work and be clear enough... and then `info_box` is something with these features...